cmd/osbuild-worker/jobimpl-osbuild.go

```diff
@@ -236,8 +236,16 @@ func (impl *OSBuildJobImpl) Run(job worker.Job) error {
 			Success: false,
 		},
 		UploadStatus: "failure",
+		Arch:         common.CurrentArch(),
 	}
 
+	hostOS, err := common.GetRedHatRelease()
+	if err != nil {
+		osbuildJobResult.JobError = clienterrors.WorkerClientError(clienterrors.ErrorKojiBuild, err.Error())
```

Comment:
Yes, thanks for catching this. It is a leftover from the recent changes in the PR based on the reviews.

Reply:
I ended up using `clienterrors.ErrorBuildJob` because it looked as the most appropriate error from the list and I didn't want to add a new error type.